### DISCUSSION TITLE
How do you Parse Symbols

### DISCUSSION THREAD

**Discussion by olakusibe** [UglyToad/PdfPig#419] [Q&A]
I have a PDF which has text, but after extracting text, some part are in text format the rest are in symbols (see screen shot below)
How do I parse this symbols to text ?
![2022-01-31 (1)](https://user-images.githubusercontent.com/6893865/151948979-a700c84e-260d-4342-9e3c-6d1772b94504.png)

**Comment by adscott1982** [comment]
I have been having the same issue, the problem isn't specific to PdfPig - every library I have tried has this issue. It seems to occur for me with text that has non-ASCII characters.

The problem is referenced here:

https://stackoverflow.com/questions/8039423/pdf-data-extraction-gives-symbols-gibberish

I have a solution of rendering the PDF to images, then performing OCR, however it is not 100% reliable.
